cookbooks/aws-parallelcluster-common/resources/efs/efs_ubuntu1804.rb

```diff
@@ -16,7 +16,10 @@
 provides :efs, platform: 'ubuntu', platform_version: '18.04'
 unified_mode true
 
-use 'partial/_build_install_efs_utils_debian'
+use '../partial/_get_package_version_rpm'
+use '../partial/_get_package_version'
```

Comment:
I think there is something wrong here, we should import the `get_package_version_dpkg` instead, or in any case the `get_package_version` cannot work.

Reply:
Thank you for spotting it. I will review the test, since it is green despite the error.

Reply:
Hard to test with ChefSpec, but I tested on EC2: working now.